dot-print/index.html

```diff
@@ -1,91 +1,91 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Welcome to Vue</title>
+        <title>Handwriting practice paper</title>
+        <meta name="description" content="Make full-page custom handwriting worksheets in seconds!"/>
         <script src="https://unpkg.com/vue"></script>
-        <link href="fonts/stylesheet.css" rel="stylesheet">
+        <link href="stylesheet.css" rel="stylesheet">
+        <link href="stylesheet.print.css" rel="stylesheet">
     </head>
     <body>
         <div id="app">
             <div id="header">
-                <h1>Dot Printer</h1>
+                <h1>Handwriting practice paper</h1>
+                <!--googleoff: index-->
+                <p id="sorrynotresponsive">Sorry! You'll need to use a device with a larger screen for this to work.</p>
+                <!--googleon: index-->
+                <p>Make full-page custom handwriting worksheets in seconds!</p>
+                <ul id="instructions">
+                    <li><strong>STEP 1:</strong> Select a writing style: <i>cursive</i> or <i>d'nealian</i></li>
```

Comment:
Text here does not match the options in the drop down

Reply:
👍  done ... d'nealean is apparently the standard way of teaching